### ISSUE TITLE
Band Not Working in VRC

### ISSUE THREAD

**Issue by Pinkuboo** [uzair-ashraf/vrc-osc-discord-band#20]
Hi, so I've followed all the steps properly and even had my friends message and call me to test it. The animations work perfectly in Unity, and in VRC, the parameters are detected, but it won't do any animations ingame for some reason.

**Comment by shadorki** [comment]
Things to recheck:
- The automated script in Unity generates a new FX layer and doesn't modify your old one, did you add your new FX layer to your avatar?
- Do you have notifications enabled for discord on your computer?

Debugging process:
- Start VRChat
- Start the application for notifications
- When you receive a notification, do you see the notification pop up on windows? that little square box you'd normally see on any notification
- Head over to your action menu, config, OSC, is OSC enabled?
- Open the OSC debug window, do you see little blue squares when you recieve a notification? If you don't wizard added a guide on how to clear your cache on your machine for VRChat, that should show the data being sent over
- Open your avatar parameters debug window, do you see the parameters available on your avatar? and are they being toggled?

**Comment by Pinkuboo** [comment]
> Things to recheck:
> 
> * The automated script in Unity generates a new FX layer and doesn't modify your old one, did you add your new FX layer to your avatar?
> * Do you have notifications enabled for discord on your computer?
> 
> Debugging process:
> 
> * Start VRChat
> * Start the application for notifications
> * When you receive a notification, do you see the notification pop up on windows? that little square box you'd normally see on any notification
> * Head over to your action menu, config, OSC, is OSC enabled?
> * Open the OSC debug window, do you see little blue squares when you recieve a notification? If you don't wizard added a guide on how to clear your cache on your machine for VRChat, that should show the data being sent over
> * Open your avatar parameters debug window, do you see the parameters available on your avatar? and are they being toggled?

Yes I've checked and tried all of this, the squares do light up and the notification does show up on my desktop, but it just doesn't transfer to the animations on the band for some reason, I'm very lost and confused.

**Comment by BreakMyChoker** [comment]
> > Things to recheck:
> > 
> > * The automated script in Unity generates a new FX layer and doesn't modify your old one, did you add your new FX layer to your avatar?
> > * Do you have notifications enabled for discord on your computer?
> > 
> > Debugging process:
> > 
> > * Start VRChat
> > * Start the application for notifications
> > * When you receive a notification, do you see the notification pop up on windows? that little square box you'd normally see on any notification
> > * Head over to your action menu, config, OSC, is OSC enabled?
> > * Open the OSC debug window, do you see little blue squares when you recieve a notification? If you don't wizard added a guide on how to clear your cache on your machine for VRChat, that should show the data being sent over
> > * Open your avatar parameters debug window, do you see the parameters available on your avatar? and are they being toggled?
> 
> Yes I've checked and tried all of this, the squares do light up and the notification does show up on my desktop, but it just doesn't transfer to the animations on the band for some reason, I'm very lost and confused.

same here, having issues with it, shows up but the band does not change animations when message/calls come in.

**Comment by FoxFireVR** [comment]
> > > Things to recheck:
> > > 
> > > * The automated script in Unity generates a new FX layer and doesn't modify your old one, did you add your new FX layer to your avatar?
> > > * Do you have notifications enabled for discord on your computer?
> > > 
> > > Debugging process:
> > > 
> > > * Start VRChat
> > > * Start the application for notifications
> > > * When you receive a notification, do you see the notification pop up on windows? that little square box you'd normally see on any notification
> > > * Head over to your action menu, config, OSC, is OSC enabled?
> > > * Open the OSC debug window, do you see little blue squares when you recieve a notification? If you don't wizard added a guide on how to clear your cache on your machine for VRChat, that should show the data being sent over
> > > * Open your avatar parameters debug window, do you see the parameters available on your avatar? and are they being toggled?
> > 
> > 
> > Yes I've checked and tried all of this, the squares do light up and the notification does show up on my desktop, but it just doesn't transfer to the animations on the band for some reason, I'm very lost and confused.
> 
> same here, having issues with it, shows up but the band does not change animations when message/calls come in.

same for me , same issues as aboves

**Comment by WolfgangAkita** [comment]
Hello, did you get an answer or a solution to the problem? I'm in the same situation as you at the moment, I've managed to do everything unfortunately apart from the blue squares in the "OSC Debug" nothing happens, I thought it would be because my avatar is in "TEST" but unfortunately even published it doesn't work, yet the executable for the OSC is active and detects that something is happening! OSC is also activated and reset...

**Comment by FoxFireVR** [comment]
> Hello, did you get an answer or a solution to the problem? I'm in the same situation as you at the moment, I've managed to do everything unfortunately apart from the blue squares in the "OSC Debug" nothing happens, I thought it would be because my avatar is in "TEST" but unfortunately even published it doesn't work, yet the executable for the OSC is active and detects that something is happening! OSC is also activated and reset...

nope still does nothing
